ui/src/table.tsx

```diff
@@ -392,6 +397,22 @@ const
       onRenderItemColumn = (item?: Fluent.IObjectWithKey & Dict<any>, _idx?: U, col?: WaveColumn) => {
         if (!item || !col) return <span />
 
+        // HACK: prevent Safari from showing a default tooltip - https://github.com/microsoft/fluentui/issues/13868
+        const hostStyles: Partial<ITooltipHostStyles> = {
+          root: {
+            '::after': {
+              content: '',
+              display: 'block',
+            }
+          },
+        }
+
+        const TooltipWrapper = ({ children }: { children: string }) => {
+          if (col.overflow === 'tooltip') return <TooltipHost styles={hostStyles} id={item.key as string} content={children} overflowMode={TooltipOverflowMode.Parent} title={children}>{children}</TooltipHost>
+          if (col.overflow === 'wrap') return <span style={{ whiteSpace: col.overflow === 'wrap' ? 'normal' : undefined }}>{children}</span>
```

Comment:
We already know that overflow === 'wrap' so:
```suggestion
          if (col.overflow === 'wrap') return <span style={{ whiteSpace: 'normal' }}>{children}</span>
```